src/components/serviceRequest/ServiceRequestDialog.tsx

```diff
@@ -75,8 +75,8 @@ function ServiceRequestDialog(props: ServiceRequestDialogProps) {
         {
           ...serviceInformationState,
           // TODO wire up lookup controls for these fields
-          animal_id: null,
```

Comment:
Can you elaborate on switching from null to ''? makes sense to me, but I was just curious on your reasons here

Reply:
To avoid this error:
https://stackoverflow.com/questions/37427508/react-changing-an-uncontrolled-input#37427579